explorer/src/components/Domain/DomainCards.tsx

```diff
@@ -48,7 +44,7 @@ export const DomainCards: FC = () => {
   const cards = useMemo<CardData[]>(() => {
     if (loading || error || !data) return []
 
-    return data.domain.map((domain) => ({
+    return data.staking_domains.map((domain) => ({
       title: domain.name,
       description: domain.name === 'nova' ? 'EVM domain' : 'Identity domain',
       href: `/${network}/${domain.name === 'nova' ? Routes.nova : Routes.autoid}`,
```

Comment:
Something doesnt' work fine here. domain.name === 'nova' is False for Nova domain. Maybe case sensitiveness. 
<img width="291" alt="image" src="https://github.com/user-attachments/assets/c04c272c-9794-4793-be05-29aa59d7bb36">

Reply:
Great observation, another example of me going back to fix something in the indexer logic and not going back to fix things in fe 🤦